sonar-core/src/main/java/org/sonar/core/user/UserDto.java

```diff
@@ -88,11 +97,39 @@ public String getScmAccounts() {
     return scmAccounts;
   }
 
-  public UserDto setScmAccounts(@Nullable String scmAccounts) {
-    this.scmAccounts = scmAccounts;
+  public List<String> getScmAccountsAsList() {
+    return decodeScmAccounts(scmAccounts);
+  }
+
+  /**
+   * List of SCM accounts separated by '|'
+   */
+  public UserDto setScmAccounts(@Nullable String s) {
```

Comment:
It seems that Mybatis do not need getters / setters : see "... MyBatis also supports private properties and private JavaBeans properties..." from http://mybatis.github.io/mybatis-3/sqlmap-xml.html